### ISSUE TITLE
(python3) python3 requires nonexistent python311

### ISSUE THREAD

**Issue by cam1170** [chocolatey-community/chocolatey-packages#2183]
### Chocolatey Version

1.3.1

### Chocolatey License

None

### Package Version

3.11.0 -> 3.11.2

### Current Behaviour

I was trying to upgrade python3 (dependency of python) to 3.11.2 but it requires a dependency of "python311" which doesn't currently exist, and it fails.

### Steps To Reproduce

1. Try to upgrade python
2. Unable to resolve dependency

### Environment

```markdown
- Operating System:
- PowerShell Version:
- Shell:
```

### Chocolatey Log

```shell
You have python v3.11.0 installed. Version 3.11.2 is available based on your source(s).
Progress: Downloading python3 3.11.2... 100%
python not upgraded. An error occurred during installation:
 Unable to resolve dependency 'python311 (= 3.11.2)'.
python package files upgrade failed with exit code 1. Performing other installation steps.
The upgrade of python was NOT successful.
python not upgraded. An error occurred during installation:
 Unable to resolve dependency 'python311 (= 3.11.2)'.
 Unsuccessful operation for python.
  Rollback to previous version (package files only)?([Y]es/[N]o):
```

**Comment by pauby** [comment]
@AdmiringWorm would force pushing 3.11.2 fix this and the other python issue we have (on mobile so can't link to that issue)?

**Comment by AdmiringWorm** [comment]
@pauby no, most likely not. It looks like there is a likelihood that there is something wrong with the updater.

I'm investigating what is going on right now.

**Comment by AdmiringWorm** [comment]
hmm, it looks like it did try to push 3.11.2 in the past, but my guess is that the push itself failed, but since it successfully pushed 3.12.x it thought it succeeded.

As this issue and #2141 is basically the same, I'll close this one out as a duplicate (fixing one will fix the other).

**Comment by AdmiringWorm** [comment]
Duplicate of #2141
